app-backend/backsplash-server/src/main/resources/logback.xml

```diff
@@ -9,6 +9,7 @@
   <logger name="org.http4s.blaze.channel.nio1" level="WARN"/>
   <logger name="geotrellis.server" level="WARN"/>
   <logger name="com.colisweb.tracing" level="${TRACING_LOG_LEVEL:-WARN}"/>
+  <logger name="com.zaxxer.hikari" level="${HIKARI_LOG_LEVEL:-DEBUG}"/>
```

Comment:
Should we consider defaulting this to `INFO` or `WARN` like the others, put possibly override to `DEBUG` via Docker Compose?

Reply:
Yeah, that's a good point

Reply:
Resolved in https://github.com/raster-foundry/raster-foundry/pull/5200/commits/a5ad5dad2fd639a3b5c449c8a99a5121c2b16b5d